lib/elixir_boilerplate_web/router.ex

```diff
@@ -7,9 +7,24 @@ defmodule ElixirBoilerplateWeb.Router do
     plug(:accepts, ["json"])
   end
 
+  pipeline :browser do
+    plug(:accepts, ["html", "json"])
+    plug(:fetch_session)
+    plug(:fetch_flash)
+    plug(:protect_from_forgery)
+    plug(:put_secure_browser_headers)
+    plug(:put_layout, {ElixirBoilerplateWeb.Layouts.View, :app})
+  end
+
   scope "/", ElixirBoilerplateWeb do
     pipe_through(:api)
 
-    get("/health", Health.Controller, :index)
+    get("/health", Health.Controller, :index, as: :api_health)
```

Comment:
Why are we customizing the _name helpers_ for both routes?

Reply:
Ah! I forgot to mention it in the PR description but this is a issue we have on all of our projects:

Since we use `Health.Controller` instead of `HealthController`, Phoenix can’t guess the name of the route.

Without the `:as` option:

```sh
_path  GET  /health  ElixirBoilerplateWeb.Health.Controller :index
_path  GET  /        ElixirBoilerplateWeb.Home.Controller :index 
```

With the `:as` option:

```sh
api_health_path  GET  /health  ElixirBoilerplateWeb.Health.Controller :index
      home_path  GET  /        ElixirBoilerplateWeb.Home.Controller :index
```

Maybe this could be a PR in Phoenix to warn or raise if we try to generate an empty path helper. Anyway, we need to generate valid helpers 😄

Reply:
Great explanation! I think we could avoid prefixing these routes with `api_` though. What do you think?

```suggestion
    get("/health", Health.Controller, :index, as: :health)
```